script.js

```diff
@@ -1 +1,122 @@
-console.log('This is a text written in js');
+/*****************************************************************/
+//--------converting to array and store answers/score------------//
+/*****************************************************************/
+
+// select all of quiz-choices(containers of each grouped choices)
+let containers = document.querySelectorAll(".quiz-choices");
+//convert nodeList to an array
+let arrayOfChoicesCtn = Array.prototype.slice.call(containers);
+
+
+//store scores here to be used when "get result button" is clicked
+let addArray = [];
+
+
+/*****************************************************************/
+//------------- store the answers to addArray ------------------- //
+//--------- select and deselect answers on each question -------- //
+/*****************************************************************/
+
+//select all choices/answer in each question
+let quizChoice = document.querySelectorAll(".quiz-choice");
+
+//add click event to all choices/answer
+quizChoice.forEach(item => {
+  item.addEventListener("click", function() {
```

Comment:
Would recommend using arrow functions since you already use them in other parts of the code anyways.

Reply:
I changed everything except the first click event because "this" doesn't work with the arrow function. I will research more about this later. thank you!

Reply:
How does it not work? Maybe I can help research too.